_docs/0.3/components/dataset-comparison.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: docs
+title: Compontent - Dataset Comparison
+version: '0.3'
+categories:
+    - '0.3'
+    - components
+redirect_from: /docs/components/dataset-comparison
+---
+
+#### Table of contents
+
+- [Table of contents](#table-of-contents)
+- [Features](#features)
+- [Features for spark-job](#features-for-spark-job)
+- [Constraints](#constraints)
+- [Concepts](#concepts)
+  - [Schema Alignment](#schema-alignment)
+  - [Provided Schema](#provided-schema)
```

Comment:
Capitalization seems inconsistent. (_Starting with a capital letter only and be done with it_ or _All Capitals with the Exception of Prepositions, Conjunctions, and Articles_? [Help](https://www.webucator.com/how-to/how-capitalize-headings-titles.cfm))